vizro-core/tests/unit/vizro/models/_components/test_graph.py

```diff
@@ -34,6 +36,87 @@ def standard_px_chart_with_title():
     return px.scatter(data_frame="gapminder", x="gdpPercap", y="lifeExp", title="Title")
 
 
+@pytest.fixture
+def graph_config_with_target():
+    def _graph_config_with_target(target="_main__.custom_bar"):
+        return {
+            "type": "graph",
+            "figure": {
+                "_target_": target,
+                "data_frame": "iris",
+                "x": "sepal_length",
+                "y": "sepal_width",
+            },
+        }
+
+    return _graph_config_with_target
+
+
+@pytest.fixture
+def expected_code():
+    def _expected_code(target: str):
```

Comment:
This is a super weird way to get three different expected results. Why not just this? No need for any fixtures or any switching logic. Did I miss something?
```
expected_code_1 = """...."""
expected_code_2 = """...."""
expected_code_3 = """...."""

  @pytest.mark.parametrize(
        "target,expected_code",
        [
            "scatter",expected_code_1,
            "tests.unit.vizro.models._components.test_graph.custom_scatter",,expected_code_2,
            "custom_bar",expected_code_3,
        ],
    )

```